zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/ZkClient.java

```diff
@@ -1740,27 +1707,34 @@ public Stat writeDataGetStat(final String path, Object datat, final int expected
 
   public void asyncCreate(final String path, Object datat, final CreateMode mode,
       final ZkAsyncCallbacks.CreateCallbackHandler cb) {
+    asyncCreate(path, datat, mode, cb, null);
+  }
+
+  public void asyncCreate(final String path, Object data, final CreateMode mode,
```

Comment:
I assume we should invoke this one in ZkBaseDataAccessor#createChildren. But I did not see it. 

Am I missing something here?

Reply:
@kaisun2000 You've got an eye on it. We have a discussion here regarding the new API vs new interface: https://github.com/apache/helix/pull/1066#discussion_r437912277
You may put some opinions there :)